code/modules/mob/living/carbon/human/life/handle_disabilities.dm

```diff
@@ -60,6 +60,7 @@
 						"SARAH HALE DID IT!!!", \
 						"Don't tell Chase", \
 						"not so tough now huh", \
+						"If you faggots don't like the PR's why don't you just code your own?", \
```

Comment:
>correct punctuation
>correct spelling
>correct grammar
>correct capitalization

how is this a brain damage line again?

Reply:
the spirit of the message

let me make it a little more retarded